packages/cozy-client/src/store/documents.spec.js

```diff
@@ -78,6 +84,10 @@ describe('extractAndMerge', () => {
             type: 'file'
           },
           blabla: 'new field',
+          cozyMedata: {
+            created_at: '123456',
+            updated_at: '987654'
+          },
```

Comment:
Would you mind adding another depth level e.g. `cozyMetadata.something.x` just to be safe?

Reply:
I minded ;)